### PR TITLE
[monitoring-kubernetes] ebpf-exporter fix min kernel version

### PR DESCRIPTION
**Pull request by aleksey-su**
## Description
Add control of minimal Linux kernel version >= `5.8.0` for `ebpf_exporter` and a corresponding alert.

## Why do we need it in the patch release (if we do)?

ebpf_exporter does not work on some kernels.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: monitoring-kubernetes
type: fix
summary: Add control of minimal Linux kernel version >= `5.8.0` for `ebpf_exporter` and a corresponding alert.
impact_level: default
```

### CODE REVIEW

**Comment by aleksey-su** [comment]
image

**Comment by aleksey-su** [comment]
image

**Comment by aleksey-su** [comment]
image
image
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

**Review by name212 — APPROVED** [review]

**Comment by aleksey-su** [comment]
/e2e/abort openstack;Standard;containerd;1.25 7653457621 7653457621-1-con-1-25 /sys/deckhouse-oss/install:pr7272 debian@45.12.229.4

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

File: modules/340-monitoring-kubernetes/docs/README_RU.md [hunk 1]
```diff
@@ -14,3 +14,5 @@ title: "Модуль monitoring-kubernetes"
 - случаев продолжительного превышения CPU steal;
 - состояния таблицы Conntrack на узлах;
 - подов с некорректным состоянием (как возможное следствие проблем с kubelet) и др.
+
+Для сбора метрик об избыточной нагрузке на память Linux требуется ядро версии >= 5.8.0
```
Comment (z9r5) [inline modules/340-monitoring-kubernetes/docs/README_RU.md:18]:
```suggestion
Для сбора метрик об избыточной нагрузке на память Linux требуется ядро версии >= `5.8.0`.
```

File: modules/340-monitoring-kubernetes/docs/README.md [hunk 2]
```diff
@@ -14,3 +14,5 @@ It safely collects metrics and provides a basic set of rules for monitoring of:
 - The cases of the prolonged CPU stealing.
 - The state of the Conntrack table on nodes.
 - The Pods that report an incorrect state (due to kubelet-related or other issues), etc.
+
+For collect metrics about Linux Memory Overcommitment, Linux kernel version to be >= 5.8.0
```
Comment (z9r5) [inline modules/340-monitoring-kubernetes/docs/README.md:18]:
```suggestion
To collect metrics about Linux Memory Overcommitment, Linux kernel version >= `5.8.0` is required.
```

**Review by z9r5 — APPROVED** [review]

**Comment by z9r5** [comment]
/e2e/abort openstack;Standard;containerd;1.25 7653457621 7653457621-1-con-1-25 /sys/deckhouse-oss/install:pr7272 debian@45.12.229.4

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Review by RomanenkoDenys — APPROVED** [review]

File: modules/340-monitoring-kubernetes/docs/README_RU.md [hunk 3]
```diff
@@ -14,3 +14,5 @@ title: "Модуль monitoring-kubernetes"
 - случаев продолжительного превышения CPU steal;
 - состояния таблицы Conntrack на узлах;
 - подов с некорректным состоянием (как возможное следствие проблем с kubelet) и др.
+
+Для сбора метрик об избыточной нагрузке на память Linux требуется ядро версии >= `5.8.0`.
```
Comment (MariaMakeeva) [inline modules/340-monitoring-kubernetes/docs/README_RU.md:18]:
```suggestion

Для сбора данных о чрезмерной нагрузке на память в Linux требуется ядро версии 5.8.0 или выше.
```

**Review by MariaMakeeva — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7316](https://github.com/deckhouse/deckhouse/pull/7316) to the branch [release-1.57](https://github.com/deckhouse/deckhouse/tree/release-1.57) successful!

**Pull request merged**